Menu/Menu.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity >= 0.6.0;
+
+struct MenuItem {
+    string title;
+    string description;
```

Comment:
I'd suggest to make descriptions be optional, but as I understand there is no optional types for our ABI.

Reply:
You're right.

Reply:
We've discussed the Menu Item with @eteslov and now think that there should be NO description for any button (menu item) since in regular user interfaces there is never a description for Menu buttons as it's excessively . 

In case you need some better description, add it in the Menu description itself.

Reply:
OK

Reply:
Discussed to keep it in order to support tooltips on desktop